server/services/forms/schemas/decorators/form_decorations/schema-decorations.js

```diff
@@ -33,39 +33,142 @@ module.exports = {
+                                ])
+                                .build()
+                        )
+                        .withField(
+                            Component('text', 'Address1')
+                                .withValidator('required')
+                                .withProp('label', 'Building')
+                                .withProp('visibilityConditions', [
+                                    {
+                                        'conditionPropertyName': 'OriginalChannel',
+                                        'conditionPropertyValue': 'POST'
+                                    }
+                                ])
+                                .build()
+                        )
+                        .withField(
+                            Component('text', 'Address2')
+                                .withValidator('required')
+                                .withProp('label', 'Street')
+                                .withProp('visibilityConditions', [
+                                    {
+                                        'conditionPropertyName': 'OriginalChannel',
+                                        'conditionPropertyValue': 'POST'
+                                    }
+                                ])
+                                .build()
+                        )
+                        .withField(
+                            Component('text', 'Address3')
+                                .withValidator('required')
+                                .withProp('label', 'Town or City')
+                                .withProp('visibilityConditions', [
+                                    {
+                                        'conditionPropertyName': 'OriginalChannel',
+                                        'conditionPropertyValue': 'POST'
+                                    }
+                                ])
+                                .build()
+                        )
+                        .withField(
+                            Component('text', 'Postcode')
+                                .withValidator('required')
+                                .withProp('label', 'Postcode')
+                                .withProp('visibilityConditions', [
+                                    {
+                                        'conditionPropertyName': 'OriginalChannel',
+                                        'conditionPropertyValue': 'POST'
+                                    }
+                                ])
+                                .build()
+                        )
+                        .withField(
+                            Component('dropdown', 'Country')
+                                .withValidator('required')
+                                .withProp('label', 'Country')
+                                .withProp('choices', 'COUNTRIES_CURRENT')
+                                // .withProp('defaultOptions', true)
```

Comment:
Sorry... just spotted a random commented out line  as well. Do you need it or can it be removed?